web/src/pages/HomePage/HomePage.tsx

```diff
@@ -96,8 +96,10 @@ const HomePage = () => {
             Redwood? Now&apos;s your chance!
           </p>
           <p>
-            Join the RedwoodJS Build Competition and challenge yourself to build
-            something new and creative.
+            <a href={Constants.APPLICATION_URL}>
+              Join the RedwoodJS Build Competition
+            </a>{' '}
+            and challenge yourself to build something new and creative.
```

Comment:
I think you can remove this whole section starting with "Do you have an idea..." through "...for a chance to win".

Just start body with How it Works.